.pre-commit-config.yaml

```
@@ -18,6 +18,7 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
+        args: [--use-parentheses]
```

```suggestion
        args: [--use-parentheses]
```

Why not add `use_parentheses = true` to the `isort` config in the `pyproject.toml` file instead? This would apply to using both `pre-commit` and `isort`. This change only applies when using `pre-commit`